src/server/logs/cmds/cmds.go

```diff
@@ -123,25 +124,31 @@ func Cmds(pachCtx *config.Context, pachctlCfg *pachctl.Config) []*cobra.Command
 			}
 			switch {
 			case cmd.Flag("logql").Changed:
-				if cmd.Flag("project").Changed || cmd.Flag("pipeline").Changed || cmd.Flag("datum").Changed {
+				if cmd.Flag("project").Changed || cmd.Flag("pipeline").Changed || cmd.Flag("datum").Changed || cmd.Flag("app").Changed {
 					fmt.Fprintln(os.Stderr, "only one of [--logQL | --project PROJECT --pipeline PIPELINE | --datum DATUM] may be set")
```

Comment:
Probably want to factor out this usage message at some point.

Reply:
It’s very slightly complex — `--pipeline` & `--project` are allowed together, so the straightforward approach of counting up flags and erroring if more than one is changed doesn’t work. But you’re right!